Makefile

```diff
@@ -46,9 +46,13 @@ vet:
 	go vet ./pkg/... ./cmd/...
 
 # Generate code
-generate:
+generate: conversion-gen
 	go generate ./pkg/... ./cmd/...
 
+# Generate conversion functions
+conversion-gen:
```

Comment:
Why the separate make _target_ for this instead of adding to the `generate` target?

Reply:
In case we won't need to run the task every time, dependency could be removed from the generate target